### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by MateuszPiaszczyk**
Navigation corrected :)

### CODE REVIEW

File: src/common/Navigation/styled.js
Comment (BoAneta) [inline src/common/Navigation/styled.js:1]: I think changes are good :) but there are still some mistakes:
Line 67 and 70 in styled Icon should be width: not weight:,
Line 77 and 78 are missing ; at the end
also line 78 and 79 are missing space afer :

File: src/index.js
Comment (BoAneta) [inline src/index.js:1]: this is old view of our core index.js, please update from the main branch.

The rest look great :) few small things and we can marge.

**Review by BoAneta — CHANGES_REQUESTED** [review]
There are few small things that needs to be change. But rest look good :)

File: src/common/Navigation/styled.js
Comment (MateuszPiaszczyk) [reply src/common/Navigation/styled.js:1]: thank you for the tips :) I'm correcting :)

**Review by MateuszPiaszczyk — COMMENTED** [review]
changes are made in App.js routers.js and main index. js. So index.js from App folder can be removed now :)

**Review by Dawid-Zieba — APPROVED** [review]
Cosmetic changes looks clean, now navigation should run with no problems.

**Pull request merged**